federation/client/clientset_generated/federation_internalclientset/typed/core/unversioned/fake/fake_core_client.go

```diff
@@ -46,8 +47,8 @@ func (c *FakeCore) Services(namespace string) unversioned.ServiceInterface {
 	return &FakeServices{c, namespace}
 }
 
-// GetRESTClient returns a RESTClient that is used to communicate
+// RESTClient returns a RESTClient that is used to communicate
 // with API server by this client implementation.
-func (c *FakeCore) GetRESTClient() *restclient.RESTClient {
-	return nil
+func (c *FakeCore) RESTClient() restclient.Interface {
+	return &fake.RESTClient{}
```

Comment:
Now it make sense. I could not figured that out. So even if you return non-interface typed nil, you can call its methods. Just the receiver is ``nil``. Well, is this golang-ish feature or is there other language that is capable of such unintuitive behaviour?

PR updated.